tests/test_helper/bats/cluster.bats

```diff
@@ -35,6 +35,9 @@ cluster_register_test_functions() {
     bats_test_function \
         --description "Open_vSwitch external_ids:ovn-remote addresses" \
         -- cluster_ovs_ovn_remote_addresses
+    bats_test_function \
+        --description "Ensure database connectivity" \
```

Comment:
nit: The description is bit vague/off. This test is more about ensuring that northd processes data between NB and SB